Add Context definition to the glossary (#225)
pharo-open-documentation/pharo-wiki: General/Glossary.md

It is also used by other programming languages such as [Squeak](#squeak), Newspeak and Cuis.

## Context

> A *Context* represent a program execution. It will store for example the `CompiledMethod` currently been executed, the receiver and arguments of the message that invoked this `CompiledMethod`, the temporary variables, the stack of calls until this moment of the execution, ...

Reviewer: ```suggestion
A *Context* represent a program execution. It will store for example the `CompiledMethod` being currently executed, the receiver and arguments of the message that invoked this `CompiledMethod`, the temporary variables, the stack of calls until this moment of the execution, ...
```

## Context

> A *Context* represent a program execution and store information about it. It will store for example the `CompiledMethod` currently been executed, the receiver and arguments of the message that invoked this `CompiledMethod`, the temporary variables, the stack of calls until this moment of the execution, ...

Reviewer: ```suggestion
A *Context* represent a program execution. It will store for example the `CompiledMethod` currently been executed, the receiver and arguments of the message that invoked this `CompiledMethod`, the temporary variables, the stack of calls until this moment of the execution, ...
```

> In Pharo, you can use the keyword `thisContext` to interact with the current context of your code. It will return the `Context` object from the moment of the execution.

Reviewer: ```suggestion
In Pharo, you can use the keyword `thisContext` to interact with the current context of your code. It will return the `Context` object at the moment of the execution. This object is different at each method call.
```
